.buildkite/premerge.steps.yaml

```diff
@@ -14,7 +14,7 @@ common: &common
     - "platform=windows"
     - "permission_set=builder"
     - "scaler_version=2"
-    - "queue=${CI_WINDOWS_BUILDER_QUEUE:-v3-1571386242-18ca3a2dfd1173f6-------z}"
+    - "queue=${CI_WINDOWS_BUILDER_QUEUE:-v3-1572523200-2f33678e336fc673-------z}"
```

Comment:
I wonder what this value is?

Reply:
Woops, these changes should be in their own, separate PR. Thanks for noticing! I've reverted them here.